apps/web/src/actions/user/setupAction.ts

```diff
@@ -0,0 +1,39 @@
+'use server'
+
+import { isWorkspaceCreated, strongPassword } from '@latitude-data/core'
+import db from '$/db/database'
+import { setSession } from '$/lib/auth/setSession'
+import { ROUTES } from '$/lib/routes'
+import setupService from '$/services/user/setupService'
+import { redirect } from 'next/navigation'
+import { z } from 'zod'
+import { createServerAction } from 'zsa'
+
+export const setupAction = createServerAction()
+  .input(
+    z.object({
+      name: z.string().min(1, { message: 'Name is a required field' }),
+      email: z.string().email(),
+      password: z.string().refine((pwd) => strongPassword(pwd), {
+        message:
+          'Password must be at least 8 characters, contain at least 1 uppercase letter, 1 lowercase letter, 1 number, and 1 special character (@$!%*?&).',
```

Comment:
can... we relax all this requirements? It's just bad XP imo, consult it with product 🙏🏼